src/topology/basic.lean

```diff
@@ -1,22 +1,42 @@
 /-
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl, Mario Carneiro
+Authors: Johannes Hölzl, Mario Carneiro, Jeremy Avigad
 
-Theory of topological spaces.
+# Basic theory of topological spaces.
 
-Parts of the formalization is based on the books:
-  N. Bourbaki: General Topology
-  I. M. James: Topologies and Uniformities
-A major difference is that this formalization is heavily based on the filter library.
+The main definition is the type class `topological space α` which endows a type `α` with a topology.
+Then `set α` gets predicates `is_open`, `is_closed` and functions `interior`, `closure` and
+`frontier`. Each point `x` of `α` gets a neighborhood filter `nhds x`, and relative versions
+`nhds_within x s` for every set `s` in `α`.
+
+Locally finite family of subsets of `α` are defined.
```

Comment:
I cannot really parse this sentence. Maybe a bit more can be added to explain what's going on?